src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -50,17 +50,19 @@ $pf-c-tree-view--MaxNesting: 10;
   // action
   --pf-c-tree-view__action--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-tree-view__action--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-tree-view__action--Color: var(--pf-global--icon--Color--light);
   --pf-c-tree-view__action--focus--BackgroundColor: var(--pf-global--BackgroundColor--200);
+  --pf-c-tree-view__action-item--Color: var(--pf-global--icon--Color--light);
+  --pf-c-tree-view__action-item--hover--Color: var(--pf-global--icon--Color--dark);
+  --pf-c-tree-view__action-item--focus--Color: var(--pf-global--icon--Color--dark);
 
   padding-top: var(--pf-c-tree-view--PaddingTop);
   padding-bottom: var(--pf-c-tree-view--PaddingBottom);
 }
 
-// stylelint-disable selector-max-class, selector-max-compound-selectors
+// stylelint-disable
 .pf-c-tree-view__list-item {
   &.pf-m-expanded {
-    > .pf-c-tree-view__content > .pf-c-tree-view__node > .pf-c-tree-view__node-toggle-icon > i {
+    > .pf-c-tree-view__content > .pf-c-tree-view__node > .pf-c-tree-view__node-toggle > .pf-c-tree-view__node-toggle-icon > i {
```

Comment:
can you just transform `.pf-c-tree-view__node-toggle-icon`? We don't use `<i>`'s in react.

Reply:
looks like react is swapping out the icon. I think we could just handle it with CSS so they don't have to do that.